File: src/main/java/ru/stnk/RestTestAPI/exception/registration/DelayException.java
package ru.stnk.RestTestAPI.exception.registration;

public class DelayException extends Exception {

    private Long delay = null;

    public DelayException(){}

    public DelayException(Long delay){
        this.delay = delay;
    }

    public Long getDelay() {
        return delay;
    }

    public void setDelay(long delay) {
        this.delay = delay;
    }
}

File: src/main/kotlin/ru/stnk/resttestapi/exception/registration/DelayException.kt
package ru.stnk.resttestapi.exception.registration

class DelayException : Exception {

    var delay: Long? = null
        private set

    constructor() {}

    constructor(delay: Long?) {
        this.delay = delay
    }

    fun setDelay(delay: Long) {
        this.delay = delay
    }
}
